internal/configsource/envvarconfigsource/README.md

```diff
@@ -0,0 +1,44 @@
+# Environment Variable Config Source (Alpha)
+
+Use the environmental variable config source instead of direct references to
+environment variables in the config to inject YAML fragments or to have default
+values in case the selected environment variable is undefined.
+
+## Configuration
+
+Under the `config_sources:` use `env:` or `env/<name>:` to create an
+environment variable config source. The following parameters are available to customize
+environment variable config sources:
+
+```yaml
+config_sources:
+  env:
+    # defaults is used to create a set of fallbacks in case the original env var is
+    # undefined in the environment.
```

Comment:
would it be helpful to accept a `prefix_` option? When it is defined all env vars will be prefixed with the value before a lookup is executed. Probably not much useful for modern environments though.

Reply:
Interesting idea, let me see how it looks like in a few examples.

Reply:
While convenient I got the impression that it becomes clear if we don't have the option so a text search on the YAML gets to the proper occurrences right away.